activities/domain/v1/domain.proto

```diff
@@ -6,8 +6,8 @@ option java_package = "io.spoud.sdm.activities.domain.v1";
 option go_package = "activities.domain.v1";
 
 import "google/protobuf/timestamp.proto";
-import "google/protobuf/wrappers.proto";
 import "global/selection/v1/reference.proto";
+import "hooks/domain/v1/domain.proto";
```

Comment:
I'm not a fan of mixing domains... Domains should be as sandboxed as possible.

So here you want to integrate the LogRecord?  Do we need the whole log record? Or maybe just the entity of the log record is enough, no?

Reply:
I need just a subset of the entity depending on the change type.  Im think of changing those Logrecord to a 
`oneof {
  ownerChange
  labelChange
  etc
}`. 

what you thing? will be better approach? what will be the best way to save such struct in the DB since which change type will have different props?

Reply:
In the db and backend it's not a big deal, do as you think it's best and easy. We can always refactor. It's just that with the API we have to be really careful. Because as soon as the API is out, it's difficult to change it as you don't know who's using it. That's why I would be more careful here.

As soon as you remove the import from hooks I'm fine. Do what you have to do with the fields.